straxbra/plugins.py

```diff
@@ -2,6 +2,10 @@
 import shutil
 
 import numpy as np
+from numpy import exp
```

Comment:
I'd prefer not to import one function when we literally just imported the whole package